runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/widgets/index.tsx

```diff
@@ -13,6 +14,7 @@ const customWidgets = {
     EditorWidget,
     DraftJSEditor,
     GlobalVariableSelectWidget,
+    FileDropzone: FileDropzoneWidget,
```

Comment:
nothing wrong with a widget in the name imo, thanks to it we are letting know that it is custom